src/main/homework4/DZ1/CreditCard.java

```diff
@@ -0,0 +1,39 @@
+package main.homework4.DZ1;
+//1. Создать класс CreditCard c полями номер счета, текущая сумма на счету.
+//Добавьте метод, который позволяет начислять сумму на кредитную
+//карточку.
+//Добавьте метод, который позволяет снимать с карточки некоторую
+//сумму.
+//Добавьте метод, который выводит текущую информацию о карточке (номер счета, текущая сумма на счету).
+
+import java.util.Scanner;
+
+public class CreditCard
+{
+    int account;
+    int currentAmount;
+
+    CreditCard(int account, int currentAmount)
+    {
+        this.account = account;
+        this.currentAmount = currentAmount;
+    }
+
+    void debit()
+    {
+        Scanner s = new Scanner(System.in);
+        System.out.println("Введите сумму для зачисления на карту");
+        currentAmount += s.nextInt();
+    }
+
+    void credit()
+    {
+        Scanner s = new Scanner(System.in);
+        System.out.println("Введите сумму для снятия с карты");
```

Comment:
1. Same as above
2. What if we decide to get amount, which is bigger, than we have?